### PR TITLE
Add some details to README

### PR DESCRIPTION
**Pull request by bfournie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,26 @@ MetalKube Ironic Container
...
 
 When updated, builds are automatically triggered on https://quay.io/repository/metalkube/metalkube-ironic/
+
+This repo supports the creation of multiple containers needed when provisioning baremetal nodes with Ironic. Eventually there will be separate images for each container, but currently separate containers can share this same image with specific entry points.
+
+The following entry points are provided:
+- runironic - Starts the ironic-conductor and ironic-api processes to manage the provisioning of baremetal nodes.  Details on Ironic can be found at https://github.com/openstack/ironic.  This is the default entry point used by the Dockerfile.
+- rundnsmasq - Runs the dnmasq dhcp server to provide addresses and initiate PXE boot of baremetal nodes.  This includes a lightweight TFTP server.  Details on dnsmasq can be found at http://www.thekelleys.org.uk/dnsmasq/doc.html.
+- runhttpd - Starts the Apache web server to provide images via http for PXE boot and for deployment of the final images.
+- runmariad - Provides a database to store information associated with baremetal nodes.
```
Comment (dtantsur) [inline README.md]: probably `runmariadb`, but I don't remember

File: README.md [hunk 2]
```diff
@@ -4,3 +4,26 @@ MetalKube Ironic Container
 This repo contains the files needed to build the Ironic images used by metalkube.
 
 When updated, builds are automatically triggered on https://quay.io/repository/metalkube/metalkube-ironic/
+
+This repo supports the creation of multiple containers needed when provisioning baremetal nodes with Ironic. Eventually there will be separate images for each container, but currently separate containers can share this same image with specific entry points.
+
+The following entry points are provided:
+- runironic - Starts the ironic-conductor and ironic-api processes to manage the provisioning of baremetal nodes.  Details on Ironic can be found at https://github.com/openstack/ironic.  This is the default entry point used by the Dockerfile.
```
Comment (dtantsur) [inline README.md]: nit: github is just a mirror, I guess the canonical URL is https://docs.openstack.org/ironic/latest/

Comment (dtantsur) [reply README.md]: Also I'd prefer the lines wrapped

Comment (juliakreger) [reply README.md]: ++ We have very nice documents in a rendered form for those curious as opposed to being dropped in a code repo.

File: README.md (hunk 1 above)
Comment (bfournie) [reply README.md]: Fixed in https://github.com/metalkube/metalkube-ironic/pull/33

File: README.md (hunk 2 above)
Comment (bfournie) [reply README.md]: Fixed in https://github.com/metalkube/metalkube-ironic/pull/33

**Pull request merged**